### ISSUE TITLE
Matchers require global browser

### ISSUE THREAD

**Issue by kirill-konshin** [webdriverio/expect-webdriverio#237]
Matchers require global `browser`, but imagine I have 2 or more browsers, how to get around it?

https://github.com/webdriverio/expect-webdriverio/blob/master/src/matchers/element/toHaveElementClass.ts#L40

`browser` var is obviously global... need to have a way to supply custom browser...

**Comment by mgrybyk** [comment]
What is the use case for this?

**Comment by kirill-konshin** [comment]
Two browser windows? Chat app, or any kind of end to end testing with realtime communication.

**Comment by mgrybyk** [comment]
Are you taking about multiremote?

**Comment by kirill-konshin** [comment]
I'm talking about `BrowserObject`: 

```ts
let browser: BrowserObject = await remote(config);
```

After all, making it global fixed the issue.

**Comment by mgrybyk** [comment]
Ah, I see, it's standalone mode.

Mind rising a PR to fix it?

**Comment by kirill-konshin** [comment]
I'm not quite familiar with codebase, but I assume something like this would work?

```ts
function toHaveElementClassFn(received: WebdriverIO.Element | WebdriverIO.ElementArray, className: string, options: ExpectWebdriverIO.StringOptions = {}, browser = global.browser): any {
```

Or extend `StringOptions`:

```ts
function toHaveElementClassFn(received: WebdriverIO.Element | WebdriverIO.ElementArray, className: string, options: ExpectWebdriverIO.StringOptions = {}): any {
   const {browser = global.browser, ...safeOptions} = options;
```

But then you'll have to have a type for all functions where `browser` might be passed.

**Comment by christian-bromann** [comment]
fixed in #753
